filters-share/specific_ELEMHIDE.txt

```diff
@@ -29,6 +29,8 @@ gamemeca.com,m.ohmynews.com,m.khan.co.kr,m.dcinside.com##div[id^="ad"]
 ! ##div[id^="AD"]
 newsworker.co.kr##div[id^="AD"]
 !
+newstomato.com##.mn_banner
```

Comment:
I cannot find the element on the website you provided.

Reply:
check mobile

Reply:
i found kiwi browser

Reply:
Found.